app/templates/join.hbs

```diff
@@ -1,2 +1,14 @@
 {{page-title 'Join'}}
-<Stepper />
\ No newline at end of file
+{{#if this.login.isLoading}}
+<div data-test-loading class='intro__loading'>
```

Comment:
Can we please center the loader here and have some min height for this page ?
Check /intro/:userId route here
![image](https://github.com/Real-Dev-Squad/website-www/assets/78433013/5b280b72-ca99-4c8f-9825-4eef96b813a7)

Reply:
done